proto/viam/app/packages/v1/packages.proto

```diff
@@ -57,7 +57,6 @@ message PackageInfo {
   PackageType type = 4;
   repeated FileInfo files = 5;
   google.protobuf.Struct metadata = 6;
-  google.protobuf.Timestamp created_on = 7;
 }
 
 message CreatePackageRequest {
```

Comment:
I think this relates to more generally thinking about changes with `Packages`, as described [here](https://viam.atlassian.net/browse/DATA-978), and we may want to discuss change uri --> url  cc @AaronCasas

Reply:
Thanks for creating a ticket from that other PR discussion, I realize I made a duplicate! I think `InternalPackage` naming makes sense, and agree on being more precise about URL. Updated.